pinecone/data/index.py

```diff
@@ -69,10 +69,10 @@
 
 def parse_query_response(response: QueryResponse, unary_query: bool):
     if unary_query:
-        response.results = None
+        response._data_store.pop("results", None)
     else:
-        response.matches = None
-        response.namespace = None
+        response._data_store.pop("matches", None)
+        response._data_store.pop("namespace", None)
     return response
```

Comment:
I don't like this at all, but I brought it back from [`main`](https://github.com/pinecone-io/pinecone-python-client/blob/main/pinecone/index.py#L69) to get some tests passing. It seems this thing is supposed to return a different data shape depending on what query parameters were used, which is a terrible design. But not something I want to address right now in this diff with all the generated changes.